plugins/GSdx/GS.h

```diff
@@ -1425,3 +1425,25 @@ enum class Filtering : uint8
 	Trilinear_Bilinear_Forced,
 	Trilinear_Always
 };
+
+//Enum order matters! Don't change it!
+enum class CRCHackLevel : int8
+{
+	Automatic = -1,
+	None = 0,
+	Minimum = 1,
+	Partial = 2,
+	Full = 3,
+	Aggressive = 4
```

Comment:
Not sure if you were addressing me or turtleli.

If you insert something (not at the end) you would shift all following values by one. E.g. somebody who selected Full would get Partial instead.

For turtleli's enum parse it would behave like you described and shouldn't be a big issue.